Sources/Workspace/Workspace.swift

```diff
@@ -432,11 +442,13 @@ public class Workspace {
     ///   - fileSystem: The file system to use, defaults to local file system.
     ///   - forRootPackage: The path for the root package.
     ///   - customToolchain: A custom toolchain.
+    ///   - strictFingerprintChecking: Indicates if fingerprint mismatch would result in warnings or failures.
     ///   - delegate: Delegate for workspace events
     public convenience init(
         fileSystem: FileSystem? = .none,
         forRootPackage packagePath: AbsolutePath,
         customToolchain: UserToolchain,
+        strictFingerprintChecking: Bool? = .none,
```

Comment:
for this initializer I think we do not need to add this option - similar to other resolver options that use the default